src/set_theory/surreal/basic.lean

```diff
@@ -75,59 +76,54 @@ theorem numeric_rec {C : pgame → Prop}
 | ⟨l, r, L, R⟩ ⟨h, hl, hr⟩ :=
   H _ _ _ _ h hl hr (λ i, numeric_rec _ (hl i)) (λ i, numeric_rec _ (hr i))
 
-theorem lt_asymm {x y : pgame} (ox : numeric x) (oy : numeric y) : x < y → ¬ y < x :=
+theorem lf_asymm {x y : pgame} (ox : numeric x) (oy : numeric y) : x ⧏ y → ¬ y ⧏ x :=
 begin
   refine numeric_rec (λ xl xr xL xR hx oxl oxr IHxl IHxr, _) x ox y oy,
   refine numeric_rec (λ yl yr yL yR hy oyl oyr IHyl IHyr, _),
-  rw [mk_lt_mk, mk_lt_mk], rintro (⟨i, h₁⟩ | ⟨j, h₁⟩) (⟨i, h₂⟩ | ⟨j, h₂⟩),
-  { exact IHxl _ _ (oyl _) (lt_of_le_mk h₁) (lt_of_le_mk h₂) },
-  { exact not_lt.2 (le_trans h₂ h₁) (hy _ _) },
-  { exact not_lt.2 (le_trans h₁ h₂) (hx _ _) },
-  { exact IHxr _ _ (oyr _) (lt_of_mk_le h₁) (lt_of_mk_le h₂) },
+  rw [mk_lf_mk, mk_lf_mk], rintro (⟨i, h₁⟩ | ⟨j, h₁⟩) (⟨i, h₂⟩ | ⟨j, h₂⟩),
+  { exact IHxl _ _ (oyl _) (move_left_lf_of_le _ h₁) (move_left_lf_of_le _ h₂) },
+  { exact not_lf.2 (le_trans h₂ h₁) (lf_of_lt (hy _ _)) },
+  { exact not_lf.2 (le_trans h₁ h₂) (lf_of_lt (hx _ _)) },
+  { exact IHxr _ _ (oyr _) (lf_move_right_of_le _ h₁) (lf_move_right_of_le _ h₂) },
 end
 
-theorem le_of_lt {x y : pgame} (ox : numeric x) (oy : numeric y) (h : x < y) : x ≤ y :=
-not_lt.1 (lt_asymm ox oy h)
+theorem le_of_lf {x y : pgame} (ox : numeric x) (oy : numeric y) (h : x ⧏ y) : x ≤ y :=
+not_lf.1 (lf_asymm ox oy h)
 
-/-- `<` is transitive when both sides of the left inequality are numeric -/
-theorem lt_trans {x y z : pgame} (ox : numeric x) (oy : numeric y) (h₁ : x < y)
-  (h₂ : y < z) : x < z :=
-lt_of_le_of_lt (le_of_lt ox oy h₁) h₂
+theorem lt_of_lf {x y : pgame} (ox : numeric x) (oy : numeric y) (h : x ⧏ y) : x < y :=
+(lt_or_fuzzy_of_lf h).resolve_right (not_fuzzy_of_le (le_of_lf ox oy h))
 
-/-- `<` is transitive when both sides of the right inequality are numeric -/
-theorem lt_trans' {x y z : pgame} (oy : numeric y) (oz : numeric z) (h₁ : x < y)
-  (h₂ : y < z) : x < z :=
-lt_of_lt_of_le h₁ (le_of_lt oy oz h₂)
+theorem lf_iff_lt {x y : pgame} (ox : numeric x) (oy : numeric y) : x ⧏ y ↔ x < y :=
+⟨lt_of_lf ox oy, lf_of_lt⟩
 
-/-- On numeric pre-games, `<` and `≤` satisfy the axioms of a partial order (even though they
-don't on all pre-games). -/
-theorem lt_iff_le_not_le {x y : pgame} (ox : numeric x) (oy : numeric y) :
-  x < y ↔ x ≤ y ∧ ¬ y ≤ x :=
```

Comment:
These theorems are all true in general, without the need of any `numeric` assumptions. In fact, they're provided automatically by the `preorder game` instance.

Reply:
I mean that you could keep them with < changed to the new ⧏ notation; those won't be consequences of `preorder`. May be useful, who knows?

Reply:
The relations `<` and `⧏` are equivalent on `numeric` games (in fact, I proved this within this PR). Since `<` implies `⧏` on all pre-games, it's preferable to use `<` over `⧏`.

Reply:
```
theorem lf_trans' {x y z : pgame} (oy : numeric y) (oz : numeric z) (h₁ : x ⧏ y) (h₂ : y ⧏ z) : x ⧏ z :=
```
This doesn't follow easily from from the `<` version, since `x` is not numeric so you can rewrite `x ⧏ y` to `x < y`, and you need to deduce `≤` from `<` from `⧏` in `h₂` and use `lf_of_lf_of_le` which seems convoluted. I agree the result is not that interesting but it seems not justified to simply remove it from mathlib.

Reply:
That theorem follows immediately from `lf_of_lf_of_le`.

Reply:
I think a better solution would be to prove `lf_of_lf_of_lt`. Again, the idea is that if you're working with numeric games, you should always prefer `<` instead of `⧏`.

Reply:
Yes, that solution and philosophy sound right to me.